hack/create_self_signed_cert.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Can we change L{4,5,6} as per my comments? Otherwise if you just run `make generate-cert` the command will be:


```
+ openssl req -newkey rsa:4096 -days 3650 -nodes -x509 -keyout '{:-/tmp/k8s-webhook-server/serving-certs}/tls.key' -out '{:-/tmp/k8s-webhook-server/serving-certs}/tls.crt' -config '{:-/tmp/k8s-webhook-server/se
rving-certs}/tls.conf'
```